spin/apps/frontend/index.html

```diff
@@ -1,31 +1,40 @@
 <html>
 <head>
-    <title>Spin Magic Eight Ball</title>
+    <title>Spin Magic AI-ght Ball</title>
```

Comment:
This changes the workshop to the assumption that everyone who does this demo has access to the AI feature, which is a limited private preview. Would it be better to keep this generic "Eight"?

Reply:
The way the workshop is laid out now, it is a requirement to complete step 3. I'll revert this.